src/plugins/vis_builder/public/saved_visualizations/_saved_vis.ts

```diff
@@ -44,7 +45,8 @@ export function createSavedVisBuilderVisClass(services: SavedObjectOpenSearchDas
           description: '',
           visualizationState: '{}',
           styleState: '{}',
-          version: 2,
+          uiState: '{}',
+          version: 3,
```

Comment:
Does this need a migration? To initially set the `uiState` for any previous VisBuilder objects?